test/mirk_tests.jl

```diff
@@ -0,0 +1,19 @@
+using SimpleBoundaryValueDiffEq, Test, BVProblemLibrary
+using LinearAlgebra
+
+for i=1:18
+    # 15 no analytical solution, 13 and 17 test problem is not right.
+    if (i==15) || (i==13) || (i==17)
```

Comment:
13th and 17th test problems are weird, the numerical and analytical solutions have deviation, but I have checked many times the implementation is correct, see [bvpT13](https://archimede.uniba.it/~bvpsolvers/testsetbvpsolvers/?page_id=212) and [bvpT17](https://archimede.uniba.it/~bvpsolvers/testsetbvpsolvers/?page_id=224). Also, SimpleBoundaryValueDiffEq.jl, BoundaryValueDiffEq.jl and bvp solvers in MATLAB behave the same in these two test problems.